guides/common/modules/proc_configuring-installation.adoc

```diff
@@ -12,6 +12,7 @@ If you lose connection to the shell where the installation command is running, s
 .Considerations
 
 * Use the `{installer-scenario} --help` command to display the available options and any default values.
```

Comment:
I wonder if this line should mention that it lists only the commonly used options, but I couldn't come up with a decent phrasing.

Reply:
```suggestion
* Use the `{installer-scenario} --help` command to display the most commonly used options and any default values.
```